packages/block/test/block.spec.ts

```diff
@@ -232,9 +184,16 @@ tape('[Block]: block functions', function (t) {
   })
 
   t.test('should test transaction validation with legacy tx in london', async function (st) {
+    // TODO: fix this test
+    // failing to initialize because mixHash is not filled with zeroes. is this a problem with
+    // the testdata, or a problem with the code?
```

Comment:
I think this is a problem with the data.  The block used in this test isn't a valid block under clique consensus rules given the mixHash has a non-zero value.